js/DisableUiComponent.js

```diff
@@ -57,16 +57,16 @@ DisableUiComponent.prototype.enable = function (target) {
 
         // Enable form elements
         $this.find(FORM_ELEMENTS)
-            .unbind('click', preventDefaultAndStopPropagation)
+            .off('click', preventDefaultAndStopPropagation)
             .removeClass('disabled')
-            .removeAttr('disabled');
+            .prop('disabled', false);
```

Comment:
We're using `prop` here in the `enable` method but setting it as an attribute in the `disable` method - see L32 (`.attr('disabled', 'disabled');`). Using a mix may cause issues. We should either set/unset the attribute or property not a mix of both.